content.js

```diff
@@ -5,7 +5,17 @@ function displayReviews(element, reviews) {
     reviewContainer.classList.add('review-container');
     reviewContainer.style.fontWeight = 'normal';
     if (reviews && reviews.rating !== undefined && reviews.reviewCount !== undefined) {
-        reviewContainer.textContent = ` ⭐${reviews.rating} (${reviews.reviewCount} reviews)`;
+        if (reviews.placeId) {
+            const link = document.createElement('a');
+            link.href = `https://www.google.com/maps/place/?q=place_id:${reviews.placeId}`;

```

Comment:
Alternatively:
```suggestion
            link.href = `https://search.google.com/local/reviews?placeid=${reviews.placeId}`;
```

- **Upside**: directly shows a popup with the reviews (0 click instead of 1 click on the reviews tab in Google Maps)
- **Downside**: opens Google search in the background rather than Google Maps, IMO it's less informative/convenient for looking at the location, if the reviews are for the right place, etc.